application-workloads/dokku/dokku-vm/azuredeploy.json

```diff
@@ -39,7 +39,7 @@
     },
     "dokkuVersion": {
       "type": "string",
-      "defaultValue": "0.28.4",
```

Comment:
So this keeps getting reverted... is there something else i should be updating to ensure the version is corrected?

Reply:
Ah, there is a bicep file for this quickstart, which is the source of truth. You should update the line in the bicep file: https://github.com/Azure/azure-quickstart-templates/blob/master/application-workloads/dokku/dokku-vm/main.bicep#L17

The `azuredeploy.json` will get automatically generated after the merge